igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import sys
+from libvirt import (
+    open as libvirt_open,
+    libvirtError,
+)
+
+
+def parse_args():
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        '--prefix', dest='prefix', type=str, default='None',
+        help='Graphite Metric Prefix')
+
+    parser.add_argument(
+        '--storagepool', dest='storagepool', type=str, default='None',
+        help='Libvirt Storage Pool Name')
+
+    parser.add_argument(
+        '--storagevol', dest='storagevol', type=str, default='None',
+        help='Libvirt Storage Volume Name')
```

Comment:
Storage pool and volume are not optional, the script won't run when they are not specified.

Reply:
they are already changed to `required=True`